Optional inputs

jpodgorni opened the pull request:
experience, diet, and other are now optional (regex changed to 0), removed last for loop for choices since the first one takes care of null checks

jpodgorni committed:
Optional inputs

experience, diet, and other are now optional (regex changed to 0), removed last for loop for choices since the first one takes care of null checks

AAlchi commented on pages/api/signup.ts:
```diff
@@ -6,18 +6,18 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
+                return res.status(400).json({ message: "Empty Required Inputs"});
             }
         }
 
```
Nice work. The only issues I saw is that when the email isn't put, it gives the email error (which is ok) + the name error (which isn't supposed to show).

The other issue is that if both name fields, for example, are blank, it gives two name errors. I think it will work if you filter it out in the loop saying that if it's already there, don't add another.

jpodgorni committed:
Optional Inputs

email in names was accident from previous test, added counter to names loop so double error does not occur
